### PR TITLE
Fixed persistent drawer that doesn't cover content

### PR DESCRIPTION
**Pull request by keanulee**
Let’s try this again… this should not cover the content when the drawer is persistent

### CODE REVIEW

File: app-drawer/app-drawer.html [hunk 1]
```diff
@@ -108,24 +126,41 @@
...
+      position: absolute;
+      top: 0;
+      bottom: 0;
       left: 0;
       right: 0;
-      bottom: -120px;
       background: var(--app-drawer-scrim-background, rgba(0, 0, 0, 0.5));
       opacity: 0;
-      transition: opacity 0.2s ease;
+      visibility: hidden;
```
Comment (frankiefu) [inline app-drawer/app-drawer.html]: Do we need to set `visibility: hidden` on #scrim?  Seems like we don't need to do that here and also L159.

Comment (keanulee) [inline app-drawer/app-drawer.html]: Yes - if not, the scrim flashes visible when toggling `app-drawer-layout` from split to narrow layout (I broke this in a previous commit). Example demo/demo6.html.

Comment (frankiefu) [inline app-drawer/app-drawer.html]: Hmm... I commented out [L136](https://github.com/PolymerLabs/app-layout/blob/fixed-drawer-v2/app-drawer/app-drawer.html#L136) and [L159](https://github.com/PolymerLabs/app-layout/blob/fixed-drawer-v2/app-drawer/app-drawer.html#L159) and then ran demo/demo6.html, I couldn't see the flashes when toggling the layout.

File: app-header/test/app-header.html [hunk 2]
```diff
@@ -48,7 +48,7 @@
   <test-fixture id="testHeader">
     <template>
       <div>
-        <app-header>
+        <app-header class="header">
```
Comment (frankiefu) [inline app-header/test/app-header.html]: Hmm... this sounds like a bug in Polymer.  Users should be able to override the :host rule.  Let me try to reproduce it in a simple testcase.

Comment (frankiefu) [inline app-header/test/app-header.html]: https://github.com/Polymer/polymer/issues/3459

**Comment by frankiefu** [comment]
lgtm

**Pull request merged**